Source/Core/InputCommon/ControllerInterface/DualShockUDPClient/DualShockUDPClient.cpp

```diff
@@ -326,7 +389,9 @@ void DeInit()
   StopHotplugThread();
 }
 
-Device::Device(Proto::DsModel model, int index) : m_model{model}, m_index{index}
+Device::Device(const std::string& name, int index, const std::string& server_address,
+               u16 server_port)
+    : m_name{name}, m_index{index}, m_server_address{server_address}, m_server_port{server_port}
```

Comment:
`name` and `server_address` can be taken by value and `std::move`d into their respective members.